### DISCUSSION TITLE
Send help!!!

### DISCUSSION THREAD

**Discussion by ghost** [pmd/pmd#4598] [Q&A]
While I try apex PMD in VS Code and it send me error

Analyzing extension-output-chuckjonas.apex-pmd-#1-Apex PMD
PMD Command: java -cp "D:\Dowload\pmd-dist-7.0.0-rc3-bin\lib\*;c:\Users\ACER\Desktop\demo\*" net.sourceforge.pmd.PMD -f csv --no-cache -d "extension-output-chuckjonas.apex-pmd-#1-Apex PMD" -R "C:\Users\ACER\Desktop\demo\target\rules.xml"
stderr:Error: Could not find or load main class net.sourceforge.pmd.PMD
Caused by: java.lang.ClassNotFoundException: net.sourceforge.pmd.PMD

**Comment by jsotuyod** [comment]
It seems that the VS code plugin you are using (not owned by us) is not yet compatible with pmd 7. You should report the issue directly to the maintainer of the plugin / ensure you are using the latest version from them.
